### PR TITLE
Grid image generation fixes

### PR DESCRIPTION
**Pull request by JustOxlamon**
- Added information about simple templating to read.me
- Fixed grid image generation (one column problem)
- Added `I` symbol detection to auto prompt matrix, and auto trim `@` if no matrix input detected

### CODE REVIEW

**Review by hlky — CHANGES_REQUESTED** [review]
Just some conflicts

File: webui.py [hunk 1]
```diff
@@ -645,7 +632,7 @@ def classToArrays( items ):
     all_prompts, prompt_matrix_parts = classToArrays(getmatrix( prompt ))
     n_iter = math.ceil(len(all_prompts) / batch_size)
     all_seeds = len(all_prompts) * [seed]
-    return all_seeds, n_iter, prompt_matrix_parts, all_prompts
```
[the remark this answers is not in the document]
Comment (JustOxlamon) [reply webui.py:648]: It reserve is for future grid calculations. 
frows will be a force_n_rows

File: webui.py [hunk 2]
```diff
@@ -671,12 +658,20 @@ def process_images(
...
+
+    if not ("|" in prompt) and prompt.startswith("@"):
+        prompt = prompt[1:]
+
     comments = []
 
     prompt_matrix_parts = []
     if prompt_matrix:
         if prompt.startswith("@"):
-            all_seeds, n_iter, prompt_matrix_parts, all_prompts = oxlamon_matrix(prompt, seed, batch_size)
```
[the remark this answers is not in the document]
Comment (JustOxlamon) [reply webui.py:679]: It reserve is for future grid calculations. 
frows will be a force_n_rows

File: webui.py [hunk 3]
```diff
@@ -849,21 +844,21 @@ def process_images(
...
                         grid = draw_prompt_matrix(grid, width, height, prompt_matrix_parts)
                     except:
                         import traceback
                         print("Error creating prompt_matrix text:", file=sys.stderr)
                         print(traceback.format_exc(), file=sys.stderr)
-                output_images.insert(0, grid)
             else:
                 grid = image_grid(output_images, batch_size)
+ 
+            if grid:
```
[the remark this answers is not in the document]
Comment (JustOxlamon) [reply webui.py:860]: i think we can just add something like

if output images more than one we can add grid image

if len(output_images) > 1: 
    if (prompt_matrix or not skip_grid) ......

**Review by JustOxlamon — COMMENTED** [review]
Resolve some conflicts as indended

**Review by hlky — APPROVED** [review]
Thanks, should be good

**Pull request merged**